packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -155,11 +151,13 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
         _outerZones.add(this);
 
         if (windowElement && _outerZones.size === 1) {
-          _disposeGlobalKeyDownListener = on(windowElement, 'keydown', this._onKeyDownCapture, true);
+          windowElement.addEventListener('keydown', this._onKeyDownCapture, true);
         }
       }
 
-      this._disposables.push(on(root, 'blur', this._onBlur, true));
+      if (windowElement) {
+        windowElement.addEventListener('blur', this._onBlur, true);
```

Comment:
This event listener needs to be unbound or it will introduce a memory leak.

Reply:
Ah yes sorry did a removeEventListener for `keydown` but not blur - just added it.